app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,32 @@
+class LikesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @like = current_user.likes.build(like_params)
+    @like.article_id = params[:article_id]
+
+    if @like.save
+      redirect_to article_path(@like.article), notice: 'Liked :)'
+    else
+      redirect_to article_path(@like.article), alert: @like.errors.full_messages.to_sentence
+    end
+  end
+
+  def destroy
+    @article = Article.find(params[:article_id])
```

Comment:
Find will give error if Article against any id is not found. Please do error Handling.